api/types/accesslist/accesslist_test.go

```diff
@@ -381,3 +381,184 @@ func TestAccessList_setInitialReviewDate(t *testing.T) {
 		})
 	}
 }
+
+func TestAccessList_IsEqual(t *testing.T) {
```

Comment:
It would be good to add a check if the ephemeral tests are not wiped out in the original structs after calling IsEqual